packages/create-pantheon-decoupled-kit/src/templates/partials/nextjs-shared/nextjsLayoutCSS.hbs

```diff
@@ -28,3 +28,18 @@
 .footerCopy > a:hover {
 	color: var(--light-blue);
 }
+
+{{#if search}}
+.searchHeaderContainer {
+    display: flex; 
+    margin-left: auto;
+    margin-right: auto;
+    flex-direction: row; 
+    flex-wrap: wrap; 
+    justify-content: center; 
+}
+
+.searchHeaderContainer > div {
+    padding-top: 2.25rem; 
```

Comment:
Use the css vars for sizes. They should be equivilent to the tailwind sizes so `margin-bottom: var(--4)` is equal to `mb-4` for example.